Makefile

```diff
@@ -377,7 +377,7 @@ foss-checks:
 ###############################################################################
 .PHONY: ci
 ## Run what CI runs
-ci: clean format-check validate-gen-versions image-all test dirty-check test-crds
+ci: clean format-check image-all test dirty-check test-crds
```

Comment:
Temporarily disable validations. Otherwise it's impossible to do a release due to circular dependency on Enterprise release in the private monorepo which in turn depends on the operator.